enterprise/security/src/main/java/org/neo4j/server/security/enterprise/auth/AuthProcedures.java

```diff
@@ -49,4 +49,29 @@ public void createUser( @Name( "username" ) String username, @Name( "password" )
         }
         shiroSubject.getUserManager().newUser( username, password, requirePasswordChange );
     }
+
+    @PerformsDBMS
```

Comment:
This is a strange name for the annotation. If expanded it'll be 'Performs Database Management System' which is confusing. It was not introduced in this PR, I understand, but please, consider changing it.